ivadomed/loader/loader.py

```diff
@@ -65,6 +64,7 @@ def load_dataset(bids_df, data_list, transforms_params, model_params, target_suf
                                 slice_axis=imed_utils.AXIS_DCT[slice_axis],
                                 transform=tranform_lst,
                                 multichannel=multichannel,
+                                subvolume_filter_fn=PatchFilter(**patch_filter_params, is_train=False if dataset_type == "testing" else True),
```

Comment:
Remove all mentions of `2D` in the [`PatchFilter` class](https://github.com/ivadomed/ivadomed/blob/b2ee8ad2b670a1aa63c5e9b980f20d3ae5d5d0ca/ivadomed/loader/patch_filter.py#L4).

Reply:
Nice catch, [fc24efd](https://github.com/ivadomed/ivadomed/pull/1164/commits/fc24efdee0bbeb46d3a0043a5a8d149adc7b88c6) should take care of this.

Reply:
Technically (if I'm not wrong), as the code is implemented, the PatchFilter would support patches of any dimensions, isn't it? Then, I would have simply removed the dimension specifications (i.e. `Discard 2D or 3D patches... -> Discard patches...`)... Or should we keep the specification since we didn't test it with 1D data even though it should work?